modelci/app/v1/endpoints/model.py

```diff
@@ -49,20 +55,23 @@ async def get_model_structure(id: str):  # noqa
 
 
 @router.patch('/structure/{id}')  # TODO: add response_model
-def update_model_structure_as_new(id: str, structure, dry_run: bool = False):  # noqa
+def update_model_structure_as_new(id: str, structure: Structure, dry_run: bool = False):  # noqa
```

Comment:
合并的时候，注意这个structure参数名换成updated_layer

Reply:
The `structure` parameter contains `layer` and `connections`, so that the data structure is the same as `update_structure`.
This is for easy migration of the frontend: just need to change the called url.